docs/semgrep-secrets/getting-started.md

```diff
@@ -159,74 +145,57 @@ Find and perform bulk operations through filtering. Perform all filter operation
 ![Secrets page and relevant triaging elements.](/img/secrets-triage.png#bordered)
 **_Figure._** Secrets page and relevant triaging elements: (a) All available filters; (b) Bulk selection toggle; (c) Bulk triage button.
 
-Triage findings in bulk by performing the following steps:
+You can triage findings in bulk by performing the following steps:
 
 1. Begin by ensuring that you display all **Open** findings.
 2. Apply filters with as much specificity as possible. You may have to perform bulk triage several times. By starting with the most specific cases, and closing the findings from those specific cases, you are able to narrow down findings as you work from specific to broad filter criteria.
-3. Click the bulk select check box.
+3. Click the bulk select check ox.
 4. Click **Triage**, then your selected triage state, typically **Ignore**.
 5. Optional: Repeat this procedure to triage all open findings.
 
-## Further customization 
+## Receive findings in through PR and MR comments
 
-* Create tickets in Jira, Linear, or Asana for secrets-related findings. See [<i class="fa-regular fa-file-lines"></i> Ticketing](semgrep-cloud-platform/ticketing/).
-* See [<i class="fas fa-external-link fa-xs"></i> Secrets API documentation](https://semgrep.dev/api/v1/docs/#tag/SecretsService).
+In addition to viewing your results in Semgrep Cloud Platform, you can set up PR or MR comments from Semgrep, which allows you to view findings-related information directly in your pull requests and merge requests.
 
-## Appendix: Semgrep Secrets filters
+![Semgrep Secrets finding in a PR comment](/img/secrets-pr-comment.png#bordered)
+**_Figure._** Semgrep Secrets finding in a PR comment.
 
-This section describes what filters are available to quickly sort through findings and perform operations on filtered findings.
+:::info
+Define which rules should be in allow, comment, or block mode in the [Policies](/semgrep-secrets/policies) page.
+:::
 
-### Triage status
+### GitHub
```

Comment:
Let's remove lines 167-189. The user is expected to have completed a core deployment and should have enabled it at this point.